pkg/provider/gke/gke.go

```diff
@@ -43,6 +43,10 @@ import (
 
 // New is the GKE constructor.
 func New(dr *provider.DeploymentResource) *GKE {
+
+	// Set NGINX_SERVICE_TYPE to LoadBalancer.
+	dr.DefaultDeploymentVars["NGINX_SERVICE_TYPE"] = "LoadBalancer"
```

Comment:
You should be setting default deployment vars here:

https://github.com/prometheus/test-infra/blob/623686db4f315d316d6ba3bce3c58dc288b14c6c/pkg/provider/provider.go#L45-L51

Reply:
Done